config/initializers/assets.rb

```diff
@@ -89,5 +89,26 @@
         ActionView::Base.assets_manifest = app.assets_manifest
       end
     end
+
+    if (webpack_manifest_file = Dir.glob("#{Rails.root}/public/webpack/manifest.json").first)
+      webpack_manifest = JSON.parse(File.read(webpack_manifest_file))
+
+      Foreman::Plugin.all.each do |plugin|
+        # Manifests may be stored under the engine installation or under the Foreman app root, then
+        # either with just the plugin name or with hyphens replaced with underscores.
+        manifest_path = File.join(plugin.path, "/public/webpack/#{plugin.id}/manifest.json")
```

Comment:
Should this be `public/...` rather than `/public/`? Or rather `File.join(plugin.path, 'public', 'webpack', plugin.id, 'manifest.json')`

Reply:
Included in https://github.com/theforeman/foreman/pull/5478